### PR TITLE
MOSIP-27134

### PR DESCRIPTION
**Pull request by ase-101**

### CODE REVIEW

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java [hunk 1]
```diff
@@ -49,6 +49,9 @@ public class ClientSettingsHelper {
 	@Autowired
 	private ClientCryptoManagerService clientCryptoManagerService;
 
+	@Value("${mosip.syncdata.regclient.support114:true}")
```
Comment (vishwa-vyom) [inline admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java:52]: Default value lets keep this false, so it avoids sending these extra data for 115 users, which is most of the countries

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/utils/SyncMasterDataServiceHelper.java [hunk 2]
```diff
@@ -1283,6 +1283,133 @@ private List<IndividualTypeDto> convertIndividualTypesToDto(List<IndividualType>
...
+				deviceDto.setMacAddress(entity.getMacAddress());
+				deviceDto.setIsDeleted(entity.getIsDeleted()==null?false:entity.getIsDeleted());
+				deviceDto.setIsActive(entity.getIsActive());
+				deviceDto.setLangCode(entity.getLangCode());
+				dtoList.add(deviceDto);
+			});
+			return dtoList;
+		}
+		return null;
+	}
```
Comment (vishwa-vyom) [inline admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/utils/SyncMasterDataServiceHelper.java:1412]: No test case changes ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/utils/SyncMasterDataServiceHelper.java (hunk 2 above)
Comment (ase-101) [reply admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/utils/SyncMasterDataServiceHelper.java:1412]: No test case changes.

File: admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java (hunk 1 above)
Comment (ase-101) [reply admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/service/helper/ClientSettingsHelper.java:52]: set it to false by default

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**